apiserver/common/action.go

```diff
@@ -185,6 +189,29 @@ func WatchOneActionReceiverNotifications(tagToActionReceiver func(tag string) (s
 	}
 }
 
+// WatchPendingActionsForReceiver to create a watcher on pending actions for one receiver.
```

Comment:
eparse?

Reply:
It's a copy paste. But I'll refactor the comments.